pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -71,6 +71,27 @@ elif command -v dnf >/dev/null 2>&1; then
 		# Workaround for https://github.com/containerd/stargz-snapshotter/issues/340
 		ln -s fusermount3 /usr/bin/fusermount
 	fi
+elif command -v yum >/dev/null 2>&1; then
+	pkgs=""
+	yum_install_flags="-y"
+	if ! rpm -ql epel-release >/dev/null 2>&1; then
+		yum install ${yum_install_flags} epel-release
+	fi
+	if ! command -v dnf >/dev/null 2>&1; then
+		pkgs="${pkgs} dnf"
```

Comment:
Why?

Reply:
Yeah, it doesn't make sense. I added to take advantage of the `elif command -v dnf` (when it would be updated). Removing.